src/Compatibility/GenAPI/Microsoft.DotNet.GenAPI/GenAPIApp.cs

```diff
@@ -18,6 +18,23 @@ namespace Microsoft.DotNet.GenAPI
     /// </summary>
     public static class GenAPIApp
     {
+        // Attributes that can work with local definition and no runtime support to light up
+        // public API behavior in the language / compiler that are not defined in all supported frameworks
+        // see https://github.com/dotnet/roslyn/blob/859f94ef2d8bf88527217bc9ad7661b6fbdf33a9/src/Compilers/Core/Portable/Symbols/Attributes/AttributeDescription.cs#L343
+        private static readonly string[] s_compilerAttributes =
+        {
+            "T:System.Runtime.CompilerServices.IsExternalInit",
+            "T:System.Runtime.CompilerServices.NullableAttribute",
+            "T:System.Runtime.CompilerServices.NullableContextAttribute",
+            "T:System.Runtime.CompilerServices.NullablePublicOnlyAttribute",
+            "T:System.Runtime.InteropServices.UnmanagedCallersOnlyAttribute",
```

Comment:
This should not be on the list. It must come from Corelib, and cannot be polyfilled.